.github/ISSUE_TEMPLATE/DeprecationTemplate.md

```diff
@@ -0,0 +1,18 @@
+name: Deprecation Notice
+about: A notice for one of few API deprecations
```

Comment:
nit/suggestion: Change to: "A notice for one or more feature or API deprecations"